backend/src/main/java/hu/flowacademy/konyhatunder/bootstrap/InitDataLoader.java

```diff
@@ -143,13 +145,13 @@ private void saveNewRecipes() {
     }
 
     private void saveNewAmountOfIngredient() {
-       List<AmountOfIngredient> savedAmountOfIngredients =  amountOfIngredientRepository.saveAll(
+        List<AmountOfIngredient> savedAmountOfIngredients = amountOfIngredientRepository.saveAll(
                 newAmountOfIngredient());
         log.info("Saved {} AmountOfIngredients", savedAmountOfIngredients.size());
     }
 
     private void saveDefaultImage() {
-        Image image = new Image("abc", "abc", new byte[0]);
+        Image image = new Image("defaultImage", "defaultImage", new byte[0]);
```

Comment:
Egy konstansban még szebb lett volna, esetleg létezhetne is az image, és akkor nem kellene félni, hogy valahol elrontja az ember. Ha már van két hely, ahol építessz arra, hogy ua értékek vannak használva, mehet a konstans (87.sor) .

Reply:
Rendben, az https://github.com/flowacademyhu/kappa-konyhatunder/issues/56 issues-n javítva lesz ez is.